server/models/migrations/next/migrate.sql

```diff
@@ -16,4 +16,14 @@ INSERT INTO unit VALUES
  * @author: mbayopanda
  * @date: 2020-06-11
  */
-DELETE FROM unit WHERE id = 162;
\ No newline at end of file
+DELETE FROM unit WHERE id = 162;
+
+/**
+ * @author: mbayopanda
+ * @date: 2020-06-23
+ */
+INSERT INTO unit VALUES
+  (271, 'Collection Capacity Report', 'TREE.COLLECTION_CAPACITY_REPORT', 'Collection Capacity Report', 144, '/reports/collectionCapacity');
```

Comment:
`unit` id 271 is already taken by `Stock Inline Movements`.  Could you assign this 272?

Reply:
Oh... Since `Stock Inline Movements` has took the place of `Stock Movement` (also the id the unit table), we don't need to add 271 for Stock Inline Movement, we must remove this line from our migrate.sql and keep 271 for Collection capacity report

Reply:
Aha, I see.  Yeah, make whatever updates are necessary to make this build 👍